xcell/mappers/mapper_P15CIB.py

```diff
@@ -0,0 +1,36 @@
+from .mapper_base_Planck import MapperBasePlanck
+import healpy as hp
+
+
+class MapperP15CIB(MapperBasePlanck):
+    def __init__(self, config):
+        """
+        config - dict
+        {'file_map': path+'COM_CompMap_Compton-SZMap-milca_2048_R2.00.fits',
+         'file_mask': path+'COM_CompMap_Compton-SZMap-masks_2048_R2.01.fits',
+         'mask_name': 'mask_CIB',
+         'nside':512}
+        """
+        self._get_Planck_defaults(config)
+
+    def _get_hm_maps(self):
+        if self.hm1_map is None:
+            hm1_map = hp.read_map(self.file_map, 1)
+            self.hm1_map = [hp.ud_grade(hm1_map,
+                            nside_out=self.nside)]
+        if self.hm2_map is None:
+            hm2_map = hp.read_map(self.file_map, 2)
+            self.hm2_map = [hp.ud_grade(hm2_map,
+                            nside_out=self.nside)]
+        return self.hm1_map, self.hm2_map
+
+    def get_mask(self):
+        if self.mask is None:
+            mask = hp.read_map(self.file_mask)
+            mask = hp.ud_grade(mask,
+                               nside_out=self.nside)
+            self.mask = mask
+        return self.mask
+
+    def get_dtype(self):
+        return 'cmbCluster_convergenceDensity_cl'
```

Comment:
There isn't a `sacc` data type for CIB, so use: https://github.com/LSSTDESC/sacc/blob/54995d2f74aed3c61d1be44a067549f2f201cc64/sacc/tracers.py#L19

Reply:
cmb_temperature

Reply:
'generic'

Reply:
Done!